src/stream.rs

```diff
@@ -300,27 +300,27 @@ pub struct RecvBuf {
     data: BinaryHeap<RangeBuf>,
 
     /// The lowest data offset that has yet to be read by the application.
-    off: usize,
+    off: u64,
 
     /// The total length of data received on this stream.
     len: usize,
```

Comment:
If `off` is `u64`, then this should be as well, as in general it is larger than `off`.

Reply:
Updated at 1116ab8